pkg/manifest/manifest_test.go

```diff
@@ -711,7 +711,7 @@ func Test_include(t *testing.T) {
 					},
 				},
 			}
-			err := m.Include(&tt.exclude, tt.includeTechPreview, &tt.profile, &tt.caps)
```

Comment:
Lets add an unit test case  when tt.includeTechPreview bool is nil. It is not a super important one but IMO we should add one to cover the use case.

Reply:
Never mind. I have added lgtm